packages/stencil-library/src/components/alert/alert.e2e.ts

```diff
@@ -1,10 +1,119 @@
-import { newE2EPage } from '@stencil/core/testing';
+import { E2EElement, E2EPage, newE2EPage } from '@stencil/core/testing';
+
+const renderAndGetAlertDiv = async (content: string): Promise<{ alertDiv: E2EElement; page: E2EPage }> => {
+  const page = await newE2EPage();
+  await page.setContent(content);
+  const alertDiv = await page.find('.usa-alert');
+  return { alertDiv, page };
```

Comment:
[sand] I don't think I see us using `page` anywhere in the tests when we call this method. Could it be removed from the return and we could just return `alertDiv` instead of an object?